proxysql/run-build-proxysql

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+ROOT_DIR=$(cd $(dirname $0)/sources/proxysql; pwd -P)
+SCRIPTS_DIR=$(cd $(dirname $0) && pwd -P)
+SOURCE_IMAGE=${1:-centos:7}
+
+docker run --rm \
+    --cap-add SYS_PTRACE \
+    --mount type=bind,source=${ROOT_DIR},destination=/tmp/proxysql \
+    --mount type=bind,source=${SCRIPTS_DIR},destination=/tmp/scripts \
+    perconalab/pxc-build:${SOURCE_IMAGE//[:\/]/-} \
+    sh -c "
+    set -o errexit
+    set -o xtrace
+    export PAT_TAG='${PAT_TAG}'
+
+    sudo bash /tmp/scripts/install-deps
```

Comment:
I do not understand why we need to do it? The main idea is to use docker images which have all needed pre-installed packages . I suggest adding all needed deps (if it does not conflict with pxc builds) to pxc-build  images or create the separate images for proxysql if needed.

Reply:
We are working on adding needed dependency packages to `pxc/docker/install-deps` file to create pxc-build images for building the ProxySQL package.